src/components/main-view/main-view.jsx

```diff
@@ -1,34 +1,31 @@
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import {MovieCard} from "../movie-card/movie-card";
 import {MovieView} from "../movie-view/movie-view"
 import { title } from "process";
 
 
 export const MainView = () => {
-    const [movies, setMovies] = useState([
-        {
-            id: 1,
-            title: "Focus",
-            image: "focus.png",
-
-            director: "Glenn Ficarra & John Requa"
-        },
-
-        {
-            id: 2,
-            title: "Zodiac",
-            image: "zodiac.png",
-            director: "David Fincher"
-        },
-        {
-            id: 3,
-            title: "Now You See Me",
-            image: "nowyouseeme.png",
-            director: "Louis Leterrier"
-        }
-    ]);
+    const [movies, setMovies] = useState([]);
 
     const [selectedMovie, setSelectedMovie] = useState(null);
+    useEffect(() => {
+        fetch("https://movie-api-7p14.onrender.com")
+        .then((response) => response.json())
+        .then((data) => {
+          const movieFromApi = data.map((movie) => { 
```

Comment:
I tried logging out (`console.log(movieFromApi)`) just to be sure if anything was returned from your API call, but I discovered all values were undefined because you were mapping to the wrong keys.

To fix this try logging out the returned value of `data` (`console.log(data)`) to see the correct keys OR look up your API schema for what should be returned.

For example `movie.title` should be `movie.Title` take note of the case.